threads.js

```diff
@@ -2036,14 +2036,32 @@ Process.prototype.reportProduct = function (a, b) {
     return +a * +b;
 };
 
+function QuotientNaNException(numerator, denominator) {
+    this.name = "Quotient exception";
+    this.message = "illegal division operation: (" + numerator + ") / (" + denominator + ")";
```

Comment:
Also, maybe something other than "illegal", too, though I'm somewhat less opposed to this. Perhaps "Unsupported"?